src/main/python/github_scraper.py

```diff
@@ -1,28 +1,47 @@
-from github import Github
+import os
+import requests
+from collections import namedtuple
+
 GITHUB_AUTH_TOKEN = 'dummy'
-github_client = Github(GITHUB_AUTH_TOKEN)
+GithubItem = namedtuple('GithubItem', 'file_name file_url repository_name repository_owner')
+
+
+def compute_directory_name(phrase):
```

Comment:
I think we should name this function differently, something like "split_with_underscore", so if we want to use it somewhere else, we can. I know, we won't, but... Just saying. I know it won't affect anything, just is a better approach, at least, in my opinion.

Reply:
i do not understand why to use name "split_with_underscore"

Reply:
When you name it that way, that function can be used in different places. You can use it to determine what a directory name should look like, or parse some other string in a different scenario. What it does it splits a string passed in using '_'. The logic inside has no clue about directories and it shouldn't.